jsx/Breadcrumbs.js

```diff
@@ -0,0 +1,44 @@
+/* exported RBreadcrumbs */
+
+/**
+ * This file contains React component for Breadcrumbs.
+ *
+ * @author Loris Team
+ * @version 1.0.0
+ *
+ */
+
+/**
+ * Breadcrumbs Component.
+ * Used for navigation on all Loris pages.
+ */
+class Breadcrumbs extends React.Component {
+
+  render() {
+    let baseURL = this.props.baseURL;
+    let breadcrumbs = this.props.breadcrumbs.map(function(element, i) {
+      let url = baseURL + element.query;
+      let crumb = (
```

Comment:
We should only use `let` when the variable needs to be reassigned. The above 4 variables should be `const` I think.